src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Details.cshtml

```diff
@@ -166,6 +166,10 @@
             <tr class="govuk-table__row">
                 <th scope="row" class="govuk-table__header das-table-cell-width-45">Name</th>
                 <td id="apprentice-name" class="govuk-table__cell das-table-cell-width-55">@Model.ApprenticeName</td>
+            </tr>
+                <tr class="govuk-table__row">
+                <th scope="row" class="govuk-table__header das-table-cell-width-45">Email</th>
+                <td id="apprentice-email" class="govuk-table__cell das-table-cell-width-55">@Model.Email</td>
```

Comment:
Couple of points I double checked on the story.

1. Label should be 'Email address' I think
2. The AT on the story says only display this line if the email address is not null